src/ts/components/subtitleoverlay.ts

```diff
@@ -319,8 +321,9 @@ class ActiveSubtitleManager {
     let id = ActiveSubtitleManager.calculateId(event);
 
     let label = new SubtitleLabel({
-      // Prefer the HTML subtitle text if set, else use the plain text
-      text: event.html || event.text,
+      // Prefer the HTML subtitle text if set, else try generating a image tag as string from the image attribute,
+      // else use the plain text
+      text: event.html || ActiveSubtitleManager.generateImageTagText(event.image) || event.text,
```

Comment:
What would happen to a subtitle event without html, but with image and text? 
Is that a reasonable use case?

Reply:
Did you read the comment @bitmovin-kenny (although it's also obvious from the IMO)?
> Prefer the HTML subtitle text if set, else try generating a image tag as string from the image attribute, else use the plain text

Reply:
I read the comment and understood the code :) I just wanted to point out, that a image + text would not work together in the same subtitle, but i guess this does not really make sense